components/mosart/bld/build-namelist

```diff
@@ -312,6 +312,7 @@ else {
     add_default($nl, 'smat_option');
     add_default($nl, 'wrmflag');
     add_default($nl, 'inundflag');
+    add_default($nl, 'use_lnd_rof_two_way');
```

Comment:
Does this line and the added line in components/mosart/bld/namelist_files/namelist_defaults_mosart.xml introduce your test namelist error?

Reply:
Yes, you are right. After removing this line, `use_lnd_rof_two_way` doesn't show in `mosart_in`, and the model still runs. But should we still keep it here to be consistent with` wrmflag`,` inundflag`, `sediflag`, etc.

Reply:
I do not think it is needed. But let us see how @hydrotian and @liho745 think about it.

Reply:
what is this flag supposed to do?

Reply:
It is used to control if land-river two-way coupling is on or off. In default, it is off. But I have tested it is not necessary to set it .false. in the default MOSART namelist. So, I have made a commit to remove it from here.

Reply:
Those kind of control mechanisms usually go in the coupler but it depends on what all has to happen to turn the coupling on.

Reply:
Hi @rljacob, it makes sense to control it on coupler. But in this development, both ELM and MOSART need to know if two-way coupling is turned on or not. But we don't know how address this without using namelist of ELM and MOSART. Do you want to us to address it in this PR?

Reply:
You put the flag in the driver namelist and then the coupler reports its value to both models via the infobuffer.  Its very error prone to rely on two different models to set the same flag.

Reply:
@rljacob ,I just made a commit to remove setting use_lnd_rof_two_way from both ELM and MOSART namelist. Instead, I put this flag into `drv_in`, and let ELM and MOSART to read if two-way coupling is turn on or not from coupler. Please let me know if that commit makes sense to you.

Reply:
Thank you, @rljacob. We just found the test doesn't pass `COMPARE_base_rest` with the default coupling configuration, such as the ELM is coupled to the coupler at 30min and MOSART at 3hr. I guess this is coupler issue, because the test passed when I set both ELM and MOSART coupled to coupler at 30 min. We are working on debugging it, will update as soon as possible.

Reply:
How long is the ERS test run for?  If you don't modify it, its a few days.  If you try to do a few timesteps, it could fail if its less then that 3 hour mosart coupling frequency.   The restart frequency has to be a multiple of the longest coupling frequency.

Reply:
It is 12 days for my ERS test run. So, it is long enough for the 3 hour mosart coupling frequency.